com.unity.multiplayer.mlapi/Runtime/Connection/NetworkClient.cs

```diff
@@ -7,6 +7,8 @@ namespace MLAPI.Connection
     /// </summary>
     public class NetworkClient
     {
+        public bool IsClientDoneLoadingScene;
```

Comment:
Please comment public's

Reply:
+1 to Matt's point.
I'd also suggest naming it as `IsLoadingScene` too (and reverse the flag being false by default).